src/execution.jl

```diff
@@ -409,7 +412,7 @@ macro btime(args...)
         $trialmin = $BenchmarkTools.minimum($trial)
         $trialallocs = $BenchmarkTools.allocs($trialmin)
         println("  ",
-                $BenchmarkTools.prettytime($BenchmarkTools.time($trialmin)),
+                $BenchmarkTools.prettytime($BenchmarkTools.realtime($trialmin)),
```

Comment:
Should we print `cputime` here as well? or `cpuratio`?

Reply:
`cpuratio` seems like a good idea